syntax = "proto3";

package chat.auth;

service AuthService {
    rpc RegisterUser (RegisterRequest) returns (RegisterResponse) {}
    rpc LoginUser (LoginRequest) returns (LoginResponse) {}
}

message RegisterRequest {
    string username = 1;
    string password = 2;
}

message RegisterResponse {
    string message = 1;
}

message LoginRequest {
    string username = 1;
    string password = 2;
}

message LoginResponse {
    string message = 1;
}
